### PR TITLE
Release branch 2.15.3

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**

### CODE REVIEW

**Comment by Pierre-Sassoulas** [comment]
@DanielNoord @jacobtylerwalls I'm considering releasing those 3 crash fixes before the astroid update, what do you think ? It's not that much more trouble.

**Comment by DanielNoord** [comment]
Fine with me. However, last minor release was on a Friday and caused considerable regressions.
I don't have a lot of time this weekend, so personally I'd prefer releasing on Monday or Tuesday in case we and the primer missed any significant regressions.

**Comment by jacobtylerwalls** [comment]
Let's wait for the astroid update.

**Comment by Pierre-Sassoulas** [comment]
I released/upgraded astroid, there's only #7476 now

**Comment by DanielNoord** [comment]
Should be able to fix that tomorrow morning!

File: doc/whatsnew/2/2.15/index.rst [hunk 1]
```diff
@@ -29,6 +29,30 @@ Marc Byrne became a maintainer, welcome to the team !
...
 
+What's new in Pylint 2.15.3?
+----------------------------
+Release date: 2022-09-19
+
+
+- Fixed a crash in the ``unhashable-member`` checker when using a ``lambda`` as a dict key.
+
+  Closes #7453 (`#7453 <https://github.com/PyCQA/pylint/issues/7453>`_)
+- Fix a crash in the ``modified-iterating-dict`` checker involving instance attributes.
```
Comment (DanielNoord) [inline doc/whatsnew/2/2.15/index.rst:40]: Didn't we fix the new lines here?

Comment (Pierre-Sassoulas) [reply doc/whatsnew/2/2.15/index.rst:40]: Not on the maintenance branch :)

**Review by DanielNoord — APPROVED** [review]

**Pull request merged**